openstack/cc3test/alerts/compute.alerts

```diff
@@ -92,3 +92,19 @@ groups:
     annotations:
       description: 'Openstack Compute Canary is down, see report for more details'
       summary: 'Openstack Compute Canary is down, see report for more details'
+
+  - alert: OpenstackComputeCanaryCreateServerFlapping
+    expr: changes(cc3test_status{service="nova",name=~"TestComputeServer_create_server.+"}[2h]) > 8
+    labels:
+      severity: info
+      tier: os
+      service: '{{ $labels.service }}'
+      context: '{{ $labels.service }}'
+      meta: 'Openstack Compute Canary is flapping for 2 hours, see report for more details'
+      dashboard: cc3test-overview?var-service={{ $labels.service }}
+      playbook: 'docs/devops/alert/{{ $labels.service }}/#test_create_server'
+      report: 'cc3test/admin/object-storage/containers/cc3test/list/reports/{{ $labels.type }}'
+    annotations:
+      description: 'Openstack Compute Canary is flapping, see last two-three report for more details'
+      summary: 'Openstack Compute Canary is flapping, see last two three report for more details'
```

Comment:
`two-three report` vs. `two three report` - shouldn't this both be the same? Also shouldn't it be report`s` if we refer to at least two?

Reply:
sure